backend/lambdas/tasks/generate_queries.py

```diff
@@ -114,9 +114,11 @@ def convert_to_col_type(val, col, table):
 
     col_type = column["Type"]
 
-    if col_type == "string" or col_type == "varchar":
+    if col_type in ["char", "string", "varchar"]:
         return str(val)
-    if col_type == "int" or col_type == "bigint":
+    if col_type in ["bigint", "int", "smallint", "tinyint"]:
```

Comment:
```suggestion
    if col_type in ("bigint", "int", "smallint", "tinyint"):
```

Nitpick. Tuples are more appropriate here.